tests/test_main.py

```diff
@@ -0,0 +1,25 @@
+from typer.testing import CliRunner
+
+runner = CliRunner()
+
+
+def test_gatorgrade_runs_correct():
+    """Test that ensures that the default command runs correctly."""
+    assert True
+
+
+def test_generate_creates_correctly():
+    """Test that ensures that the generate command creates the .yml file correctly."""
+    assert True
+
+
+def test_generate_fail_with_existing_yml():
+    """Test that ensures that a second yml file isn't generated without the force command."""
+    # result = runner.invoke(app, "generate")
+    assert True
+
+
+def test_generate_force_command_creates_yml():
```

Comment:
`force` is an option, not a command; this may be better named `test_generate_with_force_option_creates_yml`, or in the case that you're mocking the main function written by the generate team, `test_generate_with_force_option_always_calls_generate`.